renderer/views/player.js

```diff
@@ -116,6 +117,7 @@ function renderPlayerControls (state, dispatch) {
 
   // Handles a click or drag to scrub (jump to another position in the video)
   function handleScrub (e) {
+    dispatch('videoMouseMoved')
```

Comment:
@ngoldman Yeah, I think I messed up a bit with the PR. There were meant to be two seperate ones. One for the double-click to fullscreen, and the other to fix the scrub-bar from hiding when you're scrubbing.